README.md

```diff
@@ -29,7 +29,7 @@ After you're finished please remove all the comments and instructions!
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228775805-50f69904-8137-4da6-a46a-6580608a3653.png)

**[OPTIONAL]**

- [ ] It would be very great if you start your components name with a capital letter 👍 to follow the naming convention of React applications 😊 because in React names components start with a capital letter and names of other js files start with a small letter 🤔